src/components/CourseForm/CourseForm.jsx

```diff
@@ -51,32 +53,68 @@ const InputAuthor = styled.input`
 	width: 100%;
 `;
 
-const CreateCourse = ({
+const CourseForm = ({
```

Comment:
I'd recommend to create 2 new components and split logic into 2 of them: `CreateCourse` and `UpdateCourse`, which will reuse common `CourseForm` component. For your `CourseForm` component I'd add `onSubmit`, `initialValues` and `submitButtonTitle` props, which will be implemented in CreateCourse and UpdateCourse.
This will decrease the complexitiy of CourseForm component, and increase Single Responsibility principle for your app